src/main/anovos/data_analyzer/quality_checker.py

```diff
@@ -9,19 +10,32 @@
 from anovos.shared.utils import attributeType_segregation, transpose_dataframe, get_dtype
 from pyspark.sql import functions as F
 from pyspark.sql import types as T
+from pyspark.sql import SparkSession, DataFrame
 
 
-def duplicate_detection(spark, idf, list_of_cols='all', drop_cols=[], treatment=False, print_impact=False):
+def _parse_columns(list_of_cols: Union[List[str], str]):
```

Comment:
what do you think of adding delimiter or seperator part pf the function with default value |,
`def _parse_columns(list_of_cols: Union[List[str], str], delimiter="|"):`

Reply:
Is there ever a case where the delimiter is different? @VishnuGowthem